main.js

```diff
@@ -1,82 +1,190 @@
+    $choice.setAttribute('class', 'choice card-panel animated bounceInUp')
+    $choice.setAttribute('data-id', choiceId)
+    $choice.textContent = data.userChoice
+    return $choice
+  }
+}
 
-var $submitButton = document.querySelector('#submit')
-var $clearButton = document.querySelector('#clear')
-var $randomizeButton = document.querySelector('#randomize')
+function createChoiceInfo($userChoiceInput, $userNameInput) {
+  if ($userChoiceInput.value && $userNameInput.value) {
+    var choiceInfo = {
+      id: choiceId,
+      userChoice: $userChoiceInput.value,
+      userName: $userNameInput.value
+    }
+    return choiceInfo
+  }
+}
 
-var header = new Typed('.header', {
-  strings: ["Let's grab: McDonalds", "Let's grab: Burger King", "Let's grab: Pizza Hut", 'To Be Determined...'],
-  typeSpeed: 50,
-  backSpeed: 50,
-  smartBackSpace: true,
-  showCursor: false
-})
+function renderChoices(data) {
+  var $choices = []
+  for (var i = 0; i < data.length; i++) {
+    $choices.push(data[i].htmlElement)
+  }
+  return $choices
+}
 
-var slogan = new Typed('.slogan', {
-  strings: ["^10000 Can't decide? Let us decide."],
-  typeSpeed: 80,
-  showCursor: false
-})
+function fadeOutAllChoices($choicesList) {
+  var $choices = $choicesList.children
+  for (var i = 0; i < $choices.length; i++) {
+    $choices[i].setAttribute('class', 'choice card-panel animated fadeOut')
+  }
+}
 
-function renderList($unorderedList, $list) {
-  var $userChoice = document.querySelector('#user-choice')
-  var $userName = document.querySelector('#user')
-  if ($userChoice.value !== '' && $userName.value !== '') {
-    var $removeButton = document.createElement('button')
+function clearAllChoices($choicesList) {
+  while ($choicesList.firstChild) {
+    $choicesList.removeChild($choicesList.firstChild)
+  }
+}
 
-    $removeButton.setAttribute('class', 'waves-effect waves-circle waves-light btn-floating secondary-content')
-    $removeButton.setAttribute('id', 'remove')
+function createRemoveButton() {
+  var $removeButton = document.createElement('button')
+  $removeButton.setAttribute('class', 'btn-floating btn-large waves-effect waves-light red')
+  $removeButton.setAttribute('id', 'remove')
+  $removeButton.textContent = 'X'
 
-    $list.textContent = $userChoice.value
-    $removeButton.textContent = 'X'
+  return $removeButton
+}
 
-    $userChoice.value = ''
-    $userName.value = ''
+function removeChoice($choicesList, $choice) {
+  $choice.setAttribute('class', 'choice card-panel animated fadeOut')
+  setTimeout(function () {
+    $choicesList.removeChild($choice)
+  }, 1000)
+}
 
-    $unorderedList.appendChild($list)
-    $list.appendChild($removeButton)
+function randomNumber(data) {
+  if (data.length !== 0) {
+    var number = Math.round(Math.random() * (data.length - 1))
+  }
+  return number
+}
 
-    $removeButton.addEventListener('click', function (e) {
-      $unorderedList.removeChild($list)
-    })
+function renderDecision(data, number) {
```

Comment:
You used the word `render` in a function name above for a function that created and returned html elements. You used `create` for the function that created and returned an object. I think `create` would be consistent here.